index.bs

```diff
@@ -2407,11 +2408,18 @@ When this method is invoked, the user agent MUST execute the following algorithm
             <!-- this needs to be indented 2 more levels to get it to render properly -->
                 <dt id="GetAssn-ConditionalMediation-Interact-FormControl">
                   If <code>|options|.{{CredentialRequestOptions/mediation}}</code> is {{CredentialMediationRequirement/conditional}}
-                  and the user interacts with an <{input}> or <{textarea}> form control with an <{input/autocomplete}> attribute whose value
-                  contains a `"webauthn"` [=autofill detail token=],
+                  and the user interacts with an <{input}> or <{textarea}> form control with an <{input/autocomplete}> attribute whose
+                  [=non-autofill credential type=] is `"webauthn"`,
                 </dt>
+        ::  <div class="note">
+                Note: The `"webauthn"` [=autofill detail token=] must appear immediately after the last [=autofill detail token=]
+                of type "Normal" or "Contact". For example:
 
-        ::  1. If |silentlyDiscoveredCredentials| is not [=list/empty=]:
+                - `"username webauthn"`
+                - `"current-password webauthn"`
```

Comment:
I would add `autofill=` to make the examples that tiny bit clearer, but this is fine as is too.

```suggestion
                - `autofill="username webauthn"`
                - `autofill="current-password webauthn"`
```